src/System.IO.FileSystem.Watcher/ref/System.IO.FileSystem.Watcher.cs

```diff
@@ -46,6 +46,14 @@ public partial class FileSystemWatcher
         public System.IO.WaitForChangedResult WaitForChanged(System.IO.WatcherChangeTypes changeType) { return default(System.IO.WaitForChangedResult); }
         public System.IO.WaitForChangedResult WaitForChanged(System.IO.WatcherChangeTypes changeType, int timeout) { return default(System.IO.WaitForChangedResult); }
     }
+    [Serializable]
```

Comment:
>Just curious, should this type implement ISerializable?

[not according to the docs](https://msdn.microsoft.com/en-us/library/system.io.internalbufferoverflowexception(v=vs.110).aspx)